File: week8/audio_rev.m
inputfile='/home/rguktrkvalley/computational lab/week8/sample-3s.wav'
outputfile='/home/rguktrkvalley/computational lab/week8/sample-3s.wav'
[y,Fs]=audioread(inputfile);
reversedData=flipud(y);
audiowrite(outputfile,reversedData,Fs);
disp('audio is reversed and successfully saved')

File: week8/audio_rev.py
from scipy.io import wavfile
#audio format must be .wav or else cant read
(fs,x)= wavfile.read('/home/rguktrkvalley/computational lab/week7/sample-3s.wav')
# Reverse the audio sample
x_reverse = x[::-1]
wavfile.write('cash.wav', fs, x_reverse)
